.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        default: "0.0.1"
+        description: "Version of the release"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      VERSION: ${{ inputs.version }}
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: configure git
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+      - name: bump versions
+        run: |
+          DATE=$(date +"%B %d %Y")
```

Comment:
Can you stick a comment with an example formatted date please? I also think we should use YYYY-MM-DD (e.g. 2024-01-01) to keep things unambiguous